ui/src/dashboards/selectors/index.ts

```diff
@@ -14,13 +14,16 @@ export const getView = (state: AppState, id: string): View => {
 
 export const getCheckForView = (
   state: AppState,
-  {properties}: View
+  view: View
 ): Partial<Check> => {
-  if (properties.type === 'check') {
-    const checkList = state.checks.list
-    const check = checkList.find(c => c.id == properties.checkID)
+  if (view) {
```

Comment:
`_.get` could be helpful for avoiding the nested `if`s here:

```tsx
  const viewType: ViewType = get(view, 'properties.type')
  const checkID = get(view, 'properties.checkID')

  return viewType === 'check'
    ? state.checks.list.find(c => c.id === checkID)
    : null
```

Reply:
thank you that's much prettier.